doc/releases/changelog-dev.md

```diff
@@ -73,6 +73,10 @@
 * Added the `qml.TAdd` and `qml.TSWAP` qutrit operations which are the ternary analogs of the CNOT and SWAP operations respectively.
   ([#2843](https://github.com/PennyLaneAI/pennylane/pull/2843))
 
+* Added the `qml.GellMannObs` qutrit observable, which is the ternary generalization of the Pauli observables. Users must include an index as the first
```

Comment:
Optional, but what about shortening this to just `GellMann`? None of the other gates / observables specify what they are in their name.

Reply:
I think that makes sense. My reasoning was mostly arbitrary, and this would make it more consistent with the rest of the operators.